dataactvalidator/scripts/loadFile.py

```diff
@@ -141,7 +141,7 @@ def load_cfda_program(filename):
              "range_and_average_of_financial_assistance_(123)": "range_and_average_of_financial_assistance",
              "program_accomplishments_(130)": "program_accomplishments",
              "regulations__guidelines__and_literature_(140)": "regulations_guidelines_and_literature",
-             "regional_or_local_office_(151)": "regional_or_local_office",
+             "regional_or__local_office_(151)": "regional_or_local_office",
```

Comment:
This looks suspicious. When you merge down from dev, make sure to check that this is the properly named thing. Or check without whoever changed it on another branch to ensure it was supposed to be changed.

Reply:
Correct. This was necessary for the CFDA loader.